backends/cuda-ref/ceed-cuda-ref-operator.c

```diff
@@ -848,12 +848,12 @@ static int CeedOperatorApplyAddAtPoints_Cuda(CeedOperator op, CeedVector in_vec,
   CeedCallBackend(CeedGetWorkVector(ceed, impl->max_active_e_vec_len, &active_e_vec));
 
   // Get point coordinates
-  if (!impl->point_coords_elem) {
+  {
     CeedVector          point_coords = NULL;
     CeedElemRestriction rstr_points  = NULL;
 
     CeedCallBackend(CeedOperatorAtPointsGetPoints(op, &rstr_points, &point_coords));
-    CeedCallBackend(CeedElemRestrictionCreateVector(rstr_points, NULL, &impl->point_coords_elem));
+    if (!impl->point_coords_elem) CeedCallBackend(CeedElemRestrictionCreateVector(rstr_points, NULL, &impl->point_coords_elem));
     CeedCallBackend(CeedElemRestrictionApply(rstr_points, CEED_NOTRANSPOSE, point_coords, impl->point_coords_elem, request));
     CeedCallBackend(CeedVectorDestroy(&point_coords));
     CeedCallBackend(CeedElemRestrictionDestroy(&rstr_points));
```

Comment:
```suggestion
    if (!impl->point_coords_elem) CeedCallBackend(CeedElemRestrictionCreateVector(rstr_points, NULL, &impl->point_coords_elem));
    {
      uint64_t state;

      CeedCallBackend(CeedVectorGetState(point_coords, &state));
      if (impl->points_state != state) {
        CeedCallBackend(CeedElemRestrictionApply(rstr_points, CEED_NOTRANSPOSE, point_coords, impl->point_coords_elem, request));
      }
    }
    CeedCallBackend(CeedVectorDestroy(&point_coords));
    CeedCallBackend(CeedElemRestrictionDestroy(&rstr_points));
```
This will require an extra field in `CeedOperator_Cuda`, but this will prevent extra calls.